Update README.md

GuliaShubham opened the pull request:
Added YouTube Java tutorials playlist link from one of the best institutes in India to learn coding.

GuliaShubham commented:
@shahednasser please have a look I just Made the changes.

GuliaShubham commented:
@shahednasser please take a look at the resources I have put on the list.

shahednasser commented on README.md:
```diff
@@ -674,7 +675,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [SQL Tutorial - Full Database Course for Beginners - freeCodeCamp](https://youtu.be/HXV3zeQKqGY)
 - [MySQL Database - Full Course - freeCodeCamp](https://youtu.be/ER8oKX5myE0)
-
+- [MySQL tutorial](https://www.youtube.com/watch?v=u96rVINbAUI&list=PLZlA0Gpn_vH9lZBruSHgs41ldy5gZgknU)
```
This does not qualify so please remove it

shahednasser commented on README.md:
```diff
@@ -687,7 +688,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [30SecondsOfCode Node JS Snippets](https://www.30secondsofcode.org/js/t/node/p/1)
 - [The only NodeJs introduction you’ll ever need](https://codeburst.io/the-only-nodejs-introduction-youll-ever-need-d969a47ef219)
-
+- [Introduction to Node JS](https://www.youtube.com/watch?v=VShtPwEkDD0&list=PLZlA0Gpn_vH_uZs4vJMIhcinABSTUH2bY)
```
Change to the link of the playlist page

shahednasser commented on README.md:
```diff
@@ -544,7 +545,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Java Beginner Course - freeCodeCamp](https://youtu.be/7WiJGTPuVeU)
 - [Java Tutorial for Beginners - Programming with Mosh](https://youtu.be/eIrMbAQSU34)
-
+- [Java Beginners to Pro Course](https://www.youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ)
```
Change the title to the title of the playlist

GuliaShubham commented:
@shahednasser Please check now. The git tutorial is a single video and not a playlist. I changed the title of the playlist and removed the one you asked for.

shahednasser commented on README.md:
```diff
@@ -544,7 +545,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Java Beginner Course - freeCodeCamp](https://youtu.be/7WiJGTPuVeU)
 - [Java Tutorial for Beginners - Programming with Mosh](https://youtu.be/eIrMbAQSU34)
-
+- [Java + DSA + Interview Preparation Course](https://www.youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ)
```
You need to add a new line below this.

shahednasser commented on README.md:
```diff
@@ -674,7 +675,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [SQL Tutorial - Full Database Course for Beginners - freeCodeCamp](https://youtu.be/HXV3zeQKqGY)
 - [MySQL Database - Full Course - freeCodeCamp](https://youtu.be/ER8oKX5myE0)
-
+- 
```
Remove the `-` here. it should be an empty line

shahednasser commented on README.md:
```diff
@@ -687,7 +688,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [30SecondsOfCode Node JS Snippets](https://www.30secondsofcode.org/js/t/node/p/1)
 - [The only NodeJs introduction you’ll ever need](https://codeburst.io/the-only-nodejs-introduction-youll-ever-need-d969a47ef219)
-
+- [Introduction to Node.js](https://www.youtube.com/playlist?list=PLZlA0Gpn_vH_uZs4vJMIhcinABSTUH2bY)
```
Add a new empty line below this

shahednasser reviewed (changes requested):
I have a few last changes below. I'll also add the `hacktoberfest-accepted` label in case this PR doesn't get merged before the end of the day

GuliaShubham commented:
@shahednasser tried to resolve the problems please have a look.

GuliaShubham commented:
@shahednasser please have a look made some new commits and the previous is also changed.

shahednasser commented:
Great thanks!
